codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ConnectionHandlerTest.java

```diff
@@ -437,6 +445,31 @@ public void cannotSendGoAwayFrameWithIncreasingLastStreamIds() throws Exception
     }
 
     @Test
+    public void canSendGoAwayUsingVoidPromise() throws Exception {
+        handler = newHandler();
+        ByteBuf data = dummyData();
+        long errorCode = Http2Error.INTERNAL_ERROR.code();
+        handler = newHandler();
+        final Throwable cause = new ClosedChannelException();
```

Comment:
nit: we typically use something like `new RuntimeException("Fake Exception")` to signify synthetic exceptions created by tests.